packages/cursorless-engine/src/processTargets/targets/InteriorTarget.ts

```diff
@@ -12,20 +16,28 @@ export class InteriorTarget extends BaseTarget<InteriorTargetParameters> {
   insertionDelimiter = " ";
   private readonly fullInteriorRange: Range;
 
-  constructor(parameters: InteriorTargetParameters) {
-    super({
+  // TODO: don't use the constructor, instead we need to create the object with create()
+  // Are we happy about that?
+  constructor(parameters: InteriorTargetParameters & CommonTargetParameters) {
+    super(parameters);
+    this.fullInteriorRange = parameters.fullInteriorRange;
+  }
+
+  public static async create(
```

Comment:
@pokey or do we prefer this way to call async functions when constructing an object?

Reply:
this is probably better, as it avoids client needing to call initializer. I'd also make the constructor private to force them to call `create`